transcom/mymove
# B-19687 Services Counselor Tertiary Address

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19687)
## [INT PR](https://github.com/transcom/mymove/pull/13267)
## [INT PR ISSUE FIX](https://github.com/transcom/mymove/pull/13395)

## Summary

This work adds tertiary address to the services counselor screens of the office app.

It also fixes two existing issues:
- SC creating a shipment not having its secondary pickup/destination value retained
- SC creating an NTSR shipment not being prompted for a secondary delivery address

### How to test

1. Feature flag on
2. Login as a Services Counselor and locate a move that is in NEEDS COUNSELING status
3. For each shipment type, add a shipment with tertiary pickup/destination addresses
4. Confirm that the data displays on the move details page and persists on create
5. Edit the moves and remove the tertiary addresses and save
6. Edit the shipments and add tertiary addresses and confirm the data persists on edit
7. Try and break
8. Turn feature flag off
9. Confirm the data for tertiary addresses is no longer displayed, and SC is no longer given the option to add tertiary addresses on shipment create/edit
10. Confirm secondary address functionality is unaffected by tertiary address flag being turned off
11. Try and break
